platform/app/prisma/migrations/20260824120002_identifier_provider_account_id/migration.sql

```
@@ -0,0 +1,49 @@
+-- ALTER TABLE "Identifier" DROP COLUMN "providerAccountId";
+-- ALTER TABLE "Identifier" DROP COLUMN "providerId";
+
+-- AlterTable
+ALTER TABLE "Identifier" ADD COLUMN "providerAccountId" TEXT;
+
+-- AlterTable
+ALTER TABLE "Identifier" ADD COLUMN "providerId" TEXT;
+
+-- Answers an IdP callback's `(provider, subject)` lookup.
+-- CreateIndex
+CREATE INDEX "Identifier_provider_providerAccountId_idx" ON "Identifier"("provider", "providerAccountId");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)